agent/agent-tooling/src/test/java/com/microsoft/applicationinsights/agent/internal/exporter/utils/SanitizationHelperTests.java

```diff
@@ -0,0 +1,189 @@
+ * software and associated documentation files (the ""Software""), to deal in the Software
+ * without restriction, including without limitation the rights to use, copy, modify, merge,
+ * publish, distribute, sublicense, and/or sell copies of the Software, and to permit
+ * persons to whom the Software is furnished to do so, subject to the following conditions:
+ * The above copyright notice and this permission notice shall be included in all copies or
+ * substantial portions of the Software.
+ * THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR
+ * PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE
+ * FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
+ * OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ */
+
+package com.microsoft.applicationinsights.agent.internal.exporter.utils;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+public class SanitizationHelperTests {
+
+  @Test
+  public void testValidPropertiesData() {
+    // given
+    Map<String, String> properties = new HashMap<>();
+    properties.put("key1", "value1");
+    properties.put("key2", "value2");
+
+    // when
+    Map<String, String> sanitized = SanitizationHelper.sanitizeProperties(properties);
+
+    // then
+    assertThat(sanitized == properties).isTrue();
+  }
+
+  @Test
+  public void testValidMeasurementsData() {
+    // given
+    Map<String, Double> measurements = new HashMap<>();
+    measurements.put("key1", 1.0);
+    measurements.put("key2", 2.0);
+
+    // when
+    SanitizationHelper.sanitizeMeasurements(measurements);
+
+    // then
+    assertThat(measurements).containsEntry("key1", 1.0);
+    assertThat(measurements).containsEntry("key2", 2.0);
+  }
+
+  @Test
+  public void testEmptyKeyInPropertiesData() {
+    // given
+    Map<String, String> properties = new HashMap<>();
+    properties.put("", "value1");
+    properties.put("key2", "value2");
+
+    // when
+    Map<String, String> sanitized = SanitizationHelper.sanitizeProperties(properties);
+
+    // then
+    assertThat(sanitized).doesNotContainKey("");
+    assertThat(sanitized).containsEntry("empty", "value1");
+    assertThat(sanitized).containsEntry("key2", "value2");
+  }
+
+  @Test
+  public void testEmptyKeyInMeasurementsData() {
+    Map<String, Double> measurements = new HashMap<>();
+    measurements.put("", 1.0);
+    measurements.put("key2", 2.0);
+    SanitizationHelper.sanitizeMeasurements(measurements);
+    assertThat(measurements).doesNotContainKey("");
+    assertThat(measurements).containsEntry("empty", 1.0);
+    assertThat(measurements).containsEntry("key2", 2.0);
+  }
+
+  @Test
```

Comment:
please add a test for makeUniqueKey.  generate 100 same keys, and then see the result.